Python/Batch/sample4_job_scheduler.py

```diff
@@ -0,0 +1,177 @@
+# Sample4_job_scheduler.py Code Sample
+
+# Create a job schedule
+#    With job specification that has an autopool
```

Comment:
nit: this doesn't need to be indented

Reply:
All of this text is not necessary, I don't think. The other samples don't have comments to describe functionality, but they do have summaries in the README. @chloegura can you add a summary for this sample in the README too?